### ISSUE TITLE
get runtime error when compile and install

### ISSUE THREAD

**Issue by yancz1989** [apache/incubator-mxnet#9888]
I have downloaded the 1.1.0 version MXNet code from github, but after compile and install, when importing, I get following error:

RuntimeError: module compiled against API version 0xc but this version of numpy is 0xb.

After this error come out, CPU start to run at 100%.

it seems to be a numpy issue. But numpy is compiled with MKL and can be imported in python. Would any give a hand on how to solve this problem?

working on a ubuntu 16.04. since I need to use some newly developed operator, I must compile and install mxnet. Many thanks for any help!

**Comment by marcoabreu** [comment]
Hello, what version of numpy is installed on your machine? 

You can determine it with the following python code:
```
>> import numpy
>> print numpy.__version__
```

**Comment by yancz1989** [comment]
@marcoabreu I have tried both 1.13.3 and 1.14.1. they don't work either.

**Comment by marcoabreu** [comment]
Are you running on Mac by any chance?

**Comment by yancz1989** [comment]
@marcoabreu no, on a linux machine. Ubuntu 16.04, gcc 5.4, and python3.5.2.

**Comment by yancz1989** [comment]
same problem happened for v1.0.0

**Comment by yancz1989** [comment]
I found if I use "import requests" first, it would run smoothly. This should be a dependency problem, I suggest the develop team look into this. it's just a piece of advice.

**Comment by sandeep-krishnamurthy** [comment]
This PR should fix the issue - https://github.com/apache/incubator-mxnet/pull/9841
